src/parser/mod.rs

```diff
@@ -1474,6 +1474,11 @@ impl<'a> Parser<'a> {
                 let result = self.parse_comma_separated(|p| p.parse_tuple(true, true))?;
                 self.expect_token(&Token::RParen)?;
                 Ok(Expr::Rollup(result))
+            } else if self.consume_tokens(&[Token::LParen, Token::RParen]) {
```

Comment:
Thinking we can either limit the syntax to the postgres/generic dialect here if we want it postgres specific, or we can move the test case to common.rs so that the test covers all dialect?

Reply:
@iffyio Good point. The group by nothing is not only supported in Postgre, so I moved it to the common and covered all dialects that support the group by expression.